lib/handlers/domHandler.js

```diff
@@ -34,6 +34,73 @@ async function boundingBoxCenter(e) {
   };
 }
 
+async function boundingBoxRight(e) {
+  const box = await assertBoundingBox(e);
+  return {
+    x: box.x + Math.floor(box.width) - 1,
+    y: box.y + box.height / 2,
+  };
+}
+
+async function boundingBox(position = 'center', e) {
+  switch (position) {
+    case 'center':
+      return await boundingBoxCenter(e);
+    case 'topRight':
+      return await boundingBoxTopRight(e);
+    case 'topLeft':
+      return await boundingBoxTopLeft(e);
+    case 'bottomRight':
+      return await boundingBoxBottomRight(e);
+    case 'bottomLeft':
+      return await boundingBoxBottomLeft(e);
+    case 'left':
+      return await boundingBoxLeft(e);
+    case 'right':
+      return await boundingBoxRight(e);
```

Comment:
Can we add a default case too?

I assume this will be centre, in case a position has a value that is not null or any of these strings.

Reply:
@zabil https://github.com/getgauge/taiko/blob/cb130b5c75dce55e52cf0e444b201cf96ec1bd31/lib/handlers/domHandler.js#L45-L46

Reply:
I think this only handles when position is not set.

However someone could do.

```
click({position: "middle"}) // or some other string
```

We could either not click or throw a warning saying 
```
Could not find position 'middle'. Clicking at center position instead.
```